Civi/Contract/ActionProvider/Action/CreateContract.php

```diff
@@ -39,6 +39,7 @@ public function getConfigurationSpecification() {
         new Specification('default_frequency',         'Integer', E::ts('Frequency (default)'), true, 12, null, $this->getFrequencies()),
         new Specification('default_cycle_day',         'Integer', E::ts('Collection Day (default)'), false, 0, null, $this->getCollectionDays()),
         new Specification('buffer_days',               'Integer', E::ts('Buffer Days'), true, 7),
+        new Specification('multiple_contracts',         'Integer', E::ts('If a Contract already exists'), false, 0, null, $this->getMultipleContractOptions()),
```

Comment:
I think the default should be ``true``, since that's the current behaviour. Or am I wrong about this?

Reply:
I think the default should be 0 because ("ignore" / continue creating a contract even if the contact already has one).
Only if you explicitly select "show error message" (Value: 1) Contract creation should be skipped if the contact already has a contract.